### PR TITLE
(lat.ls.perseus-eng2.xml) various edits

### PR DESCRIPTION
**Pull request by nkprasad12**
See commit history for details,

Fix incorrectly set sense breaks, remove empty itype tags, fix typo, and fix two instances of merged entries

### CODE REVIEW

File: CTS_XML_TEI/perseus/pdllex/lat/ls/lat.ls.perseus-eng2.xml [hunk 1]
```diff
@@ -30401,8 +30401,8 @@ First cut at header to Lewis and Short.
...
+<entryFree key="mungo" type="hapax" id="n29985a"><orth lang="la" extent="full">mungo</orth>, <itype>ere</itype> <etym>obsolete, the orig. form whence emungo, mucus, mucere, mucor, mucedo; cf. Sanscr. muk, to let go, throw off; Gr. <foreign lang="greek">ἀπομύσσω</foreign>, to wipe away; <foreign lang="greek">μυκτήρ</foreign>, nose; <foreign lang="greek">μύξα</foreign>, snivel</etym>, <sense level="1" n="I" id="n29985.3"><hi rend="ital">to blow the nose</hi>: mungo, <foreign lang="greek">μύσσω</foreign>, Gloss. Philox.</sense></entryFree>
```
Comment (lcerrato) [inline CTS_XML_TEI/perseus/pdllex/lat/ls/lat.ls.perseus-eng2.xml:30405]: There is a sense level dependency here that should perhaps be modified:
`<sense level="1" n="I" id="n29985.3">...</sense>`
It is no longer the third sense in entry n29985 and needs to be grouped with n29985a.
I will need to research the print and the current practice to determine the correct level attributions as none of this is in front of me at present.

File: CTS_XML_TEI/perseus/pdllex/lat/ls/lat.ls.perseus-eng2.xml [hunk 2]
```diff
@@ -33280,8 +33280,8 @@ First cut at header to Lewis and Short.
...
+<entryFree key="optostrotum" type="main" id="n32852a"><orth lang="la" extent="full">optostrōtum</orth>, <itype>i</itype>, <gen>n.</gen> <etym><foreign lang="greek">ὀπτόσ-στρωτόν</foreign></etym>, <sense level="1" n="I" id="n32852.3"><hi rend="ital">a brick pavement</hi>, <bibl><author>Not. Tir.</author> p. 164</bibl>.</sense></entryFree>
```
Comment (lcerrato) [inline CTS_XML_TEI/perseus/pdllex/lat/ls/lat.ls.perseus-eng2.xml:33284]: Same as above. There is a dependent sense in the new entry that is probably now structurally inaccurate:
`<sense level="1" n="I" id="n32852.3">`

**Review by lcerrato — CHANGES_REQUESTED** [review]
@nkprasad12 Just a couple of questions about sense ids in the new entries.

File: CTS_XML_TEI/perseus/pdllex/lat/ls/lat.ls.perseus-eng2.xml (hunk 1 above)
Comment (nkprasad12) [reply CTS_XML_TEI/perseus/pdllex/lat/ls/lat.ls.perseus-eng2.xml:30405]: The original text doesn't have any sense headings here at all, but I see many digitized entries in the XML contain sense <sense level="1" n="I" id="nX.0"> for the definition part.

So in this case I would think it should be
``` 
<sense level="1" n="I" id="n29985a.0">
```
Let me know what you think.

File: CTS_XML_TEI/perseus/pdllex/lat/ls/lat.ls.perseus-eng2.xml (hunk 2 above)
Comment (nkprasad12) [reply CTS_XML_TEI/perseus/pdllex/lat/ls/lat.ls.perseus-eng2.xml:33284]: Similar to above, the original text doesn't contain any subheadings. But in keeping with the format for the digitized version, I think this should end up as 

```
<sense level="1" n="I" id="n32852a.0">
```

Let me know what you think.

File: CTS_XML_TEI/perseus/pdllex/lat/ls/lat.ls.perseus-eng2.xml (hunk 1 above)
Comment (lcerrato) [reply CTS_XML_TEI/perseus/pdllex/lat/ls/lat.ls.perseus-eng2.xml:30405]: @nkprasad12 I agree. I think only the new entry designation (the "a") is what is missing in both cases.

File: CTS_XML_TEI/perseus/pdllex/lat/ls/lat.ls.perseus-eng2.xml (hunk 2 above)
Comment (lcerrato) [reply CTS_XML_TEI/perseus/pdllex/lat/ls/lat.ls.perseus-eng2.xml:33284]: @nkprasad12 As above.

Comment (nkprasad12) [reply CTS_XML_TEI/perseus/pdllex/lat/ls/lat.ls.perseus-eng2.xml:33284]: Done.

**Comment by lcerrato** [comment]
@nkprasad12 
This is fabulous. Thank you so much for this work and your patience!

**Pull request merged**